src/main/java/edu/ksu/canvas/attendance/submitter/AssignmentSubmitter.java

```diff
@@ -75,9 +73,8 @@ public void submitCourseAttendances(boolean isSimpleAttendance, List<AttendanceS
     private void gradePushingValidation(Long courseId, OauthToken oauthToken, AttendanceAssignment assignmentConfigurationFromSetup,
                                         AttendanceSummaryModel model, AttendanceAssignment attendanceAssignment) throws AttendanceAssignmentException{
 
-        if (attendanceAssignment.getStatus() == AttendanceAssignment.Status.UNKNOWN){
             attendanceAssignment = assignmentValidator.validateConfigurationSetupExistence(model, attendanceAssignment);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Introduce a new variable instead of reusing the parameter "attendanceAssignment". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1226)